VERSION

```diff
@@ -1 +1,2 @@
-18.10.2
+!!! Auto generated by make. Update version in ess.el instead !!!
```

Comment:
We should just remove this version file. I think it's only needed by a couple doc files and we can update them with sed.

Reply:
It's a prerequisite for a few targets and it's not easy to get rid of but could be removed from git repo and ignored in .gitignore.

Reply:
Changelog, dist, essr and one RPM related target depend on VERSION file changes. We need a marker of some kind to keep track of the old version. I don't think there is a more elegant solution to this, but I am not a make expert.